pwa/app/tailwind.css

```diff
@@ -64,6 +64,10 @@
   * {
     @apply border-border;
   }
+  html,
+  body {
+    @apply scroll-pt-14 antialiased;
```

Comment:
What does this fix? Again not a blocking comment, asking because I don't know

Reply:
Sure, the `scroll-pt-14` makes the browser know that there is a header with an offset of 56px - each scrolling to an element on the page (like via `#` anchor) will be `+56px`.

For example, scrolling to `#2024isde1` before will show a cut element:
![CleanShot 2024-09-15 at 07 28 45@2x](https://github.com/user-attachments/assets/164ed3d2-31fd-4ca6-8f5c-6de34bcb3bbf)

And this is the same scrolling with the class applied:
![CleanShot 2024-09-15 at 07 30 05@2x](https://github.com/user-attachments/assets/5617ae93-50a4-489b-9ab3-f9c6110ff9ca)

Reply:
The `antialiased` makes the font look better on certain displays, like in maoOS.